apps/mull-ui/src/app/components/chat-input/chat-input.tsx

```diff
@@ -89,6 +88,11 @@ export const ChatInput = ({
             </button>
           }
         />
+        {LIMITS.POST_MESSAGE - formik.values.message.length < 100 ? (
+          <div className="chat-input-limit-message">
+            {LIMITS.POST_MESSAGE - formik.values.message.length}/{LIMITS.POST_MESSAGE}
```

Comment:
this will show a negative number if it's over the limit (i.e -1/512 instead of 513/512)

Reply:
To add on to this, the text stays until the user starts typing in the chat box again
![image](https://user-images.githubusercontent.com/33991121/113925749-61c55b80-97b9-11eb-9590-77c577b9dfb7.png)

Reply:
The negative number is intentional, a few apps do that

> the text limit stays until the user starts typing in the chat box again

Will try and fix this

Reply:
Can you show which apps do that, i'm very curious

Reply:
twitter for one
![image](https://user-images.githubusercontent.com/23544999/113975127-0cb33500-980d-11eb-8c8e-de3b9af9341d.png)

Reply:
Fixed issue that ritchelle mentionned. Was due to some desync between form values and textcontent of div.